ac_config.lua

```diff
@@ -19,6 +19,8 @@ Components = {
 	["client.duiblacklist"] = true, -- checks for certain runtime textures being created which are used in cheat menus
 	["client.bypasshacks"] = true, -- checks if their cheats try to disable certain anticheat components, not exclusive to anticheese
 	["client.esx.gcphone"] = true, -- various detections for gcphone crashers/abuse
+	["client.night"] = true, -- anti night vision
```

Comment:
any reason this was moved between some ESX settings? would probably better off being before or after them.

Reply:
to be with client.

Reply:
would probably best moving this above or below esx, just so its not "somewhere in the middle"

Reply:
Changed